app/src/org/odk/collect/android/widgets/StringWidget.java

```diff
@@ -222,4 +222,11 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
     public void setLastQuestion(boolean isLast){
            // nothing changes for Strings
     }
+
+    /**
+     * For testing purposes
+     */
+    public void setAnswer(String answer) {
+        mAnswer.setText(answer);
+    }
```

Comment:
Eventually we should make a whole form entry abstraction layer for filling out forms in a way similar to calabash. This is just a hacky start